tests/python/unittest/test_higher_order_grad.py

```diff
@@ -27,7 +27,8 @@
 from common import with_seed
 import mxnet
 from mxnet import nd, autograd, gluon
-from mxnet.test_utils import assert_almost_equal, random_arrays, rand_shape_nd, same
+from mxnet.test_utils import (
+    assert_almost_equal, random_arrays, random_uniform_arrays, rand_shape_nd, same)
```

Comment:
Do you still need import random_arrays in this case? Also can this new method replace random_arrays?

Reply:
Other tests are still using `random_arrays`, probably they can be replaced with `random_uniform_arrays` but will have to verify once.